packages/ui/.storybook/preview.js

```diff
@@ -0,0 +1,73 @@
+import '../styles/index.css';
+
+const screens = {
+  xs: '320px',
+  sm: '576px',
+  md: '768px',
+  lg: '992px',
+  xl: '1200px',
+  '2xl': '1536px',
+  '3xl': '1854px',
+};
```

Comment:
```suggestion
import '../styles/index.css';
import resolveConfig from 'tailwindcss/resolveConfig';
import tailwindConfig from '@sovryn/tailwindcss-config/index';

const config = resolveConfig(tailwindConfig);

const screens = {
  xs: '320px',
  ...config.theme.screens,
};
```

Reply:
We shouldn't hardcode Tailwind config values but use the actual config values we have defined - see https://tailwindcss.com/docs/configuration#referencing-in-java-script for reference.